.github/workflows/daily.yml

```
@@ -45,7 +45,7 @@ jobs:
           cache: pip
           cache-dependency-path: requirements-tests.txt
       - name: Install dependencies
-        run: pip install $(grep mypy== requirements-tests.txt)
+        run: pip install typing_extensions $(grep ^pathspec requirements-tests.txt)
```

Since we're switching to creating a "clean" virtual environment dynamically in this PR in which to run the test, we may as well simplify this step a little bit (and bring it in line with all our other CI jobs that `pip install` stuff). There's no longer any need to worry about the script running in an isolated environment:

```suggestion
        run: pip install -r requirements-tests.txt
```